RSFormView/Classes/Views/TextField/TextFieldView.swift

```diff
@@ -22,6 +22,10 @@ class TextFieldView: UIView {
   @IBOutlet weak var bottomLineView: UIView!
   @IBOutlet weak var textFieldContainerView: UIView!
   @IBOutlet weak var titleLabelContainerView: UIView!
+  @IBOutlet weak var labelToTextFieldDistance: NSLayoutConstraint!
+  @IBOutlet weak var textFieldToBottomLineDistance: NSLayoutConstraint!
+  @IBOutlet weak var bottomLineToErrorLabelDistance: NSLayoutConstraint!
+  @IBOutlet weak var textFieldContainerBottomToErrorLabelDistance: NSLayoutConstraint!
```

Comment:
I like the naming you used here :)

Reply:
oh, I missed this, I like the naming as well but lets keep the type as suffix: 
so `textFieldContainerBottomToErrorLabelDistance` becomes `textFieldContainerToErrorLabelConstraint ` (also lets miss the Bottom in this case)